cranelift-codegen/src/isa/x86/enc_tables.rs

```diff
@@ -1223,16 +1223,25 @@ fn convert_ineg(
     let mut pos = FuncCursor::new(func).at_inst(inst);
     pos.use_srcloc(inst);
 
+    // FIXME Is it necessary to do this? I thought the generated code only calls this function if
+    // the opcode is ineg, maybe this could just be an assert? I guess this is a very common
+    // pattern in this file, so maybe there's a good reason?
```

Comment:
This is necessary to get the arg passes to ineg. Adding an `unreachable!()` as else would be nice though.

Reply:
Added an `else { unreachable!() }` clause